client/app/domain/domain.service.js

```diff
@@ -597,7 +600,7 @@ angular.module('services').service(
             'CNAME',
             subDomain,
             entry.excludeId,
-          ).then(haveNoBrothers => haveNoBrothers);
+          ).then(haveNoBrothers => ({ recordCanBeAdded: haveNoBrothers }));
```

Comment:
Could you please rename the whole `brother` thing please ? It's neither correct English neither a technical term.

You could for example they `subDomainsWithSameFieldTypes`

Reply:
All brother things have been removed :)